packages/server/src/scim.ts

```diff
@@ -699,6 +725,96 @@ export class ScimServer implements DirectoryProvider {
         httpRes.end();
     }
 
+    private async _handleSamlPost(httpReq: IncomingMessage, httpRes: ServerResponse) {
+        let newUser: ScimUser;
+        const { secretToken, orgId } = this._getDataFromScimRequest(httpReq);
+
+        if (!secretToken || !orgId) {
+            return this._sendErrorResponse(httpRes, 400, "Empty SCIM Secret Token / Org Id");
+        }
+
+        try {
+            const body = await readBody(httpReq);
+            const params = new URLSearchParams(body);
+            const samlResponse = params.get("SAMLResponse") || "";
+            const xmlSamlResponse = Buffer.from(samlResponse, "base64").toString("ascii");
+
+            // TODO: Remove this
+            console.log(JSON.stringify({ xmlSamlResponse }, null, 2));
```

Comment:
Keeping this for now, like the other SCIM data, so we can debug in case something strange happens.